controller/PciManager.php

```diff
@@ -208,30 +223,33 @@ public function export()
         //as upload may be called multiple times, we remove the session lock as soon as possible
         session_write_close();
 
-        try{
+        try {
             if (!$this->hasRequestParameter('typeIdentifier')) {
                 throw new PortableElementException('Type identifier parameter missing.');
             }
-
             $identifier = $this->getRequestParameter('typeIdentifier');
             $path = $this->getService()->export(PciModel::PCI_IDENTIFIER, $identifier);
-            $object = $this->getService()->getPortableElementByIdentifier(PciModel::PCI_IDENTIFIER, $identifier);
-            $this->returnJson($this->getMinifiedModel($object));
-
-        } catch(\common_Exception $e) {
+            header('Content-Type: application/zip');
```

Comment:
do you need thiis line `header('Content-Type: application/zip');` since `public static function returnFile($filename, $contenttype = true,...` is supposed to serve itself the correct content type header ?